components/shared/CommonTokenMoney.vue

```diff
@@ -15,20 +15,23 @@ import Money from '@/components/shared/format/Money.vue'
 import TokenMoney from '@/components/bsx/format/TokenMoney.vue'
 import { getKusamaAssetId } from '@/utils/api/bsx/query'
 
-withDefaults(
-  defineProps<{
-    value: number | string | undefined
-    customTokenId?: string | undefined
-    dataCy: string
-    round: number
-  }>(),
-  {
-    value: '0',
-    customTokenId: '',
-    dataCy: '',
-    round: 4,
-  }
-)
+defineProps({
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-expect-error
+  value: {
+    type: [Number, String, BigInt],
+    default: '0',
+  },
+  dataCy: {
+    type: String,
+    required: false,
+    default: '',
+  },
+  round: {
+    type: Number,
+    default: 4,
+  },
+})
```

Comment:
Could we use this instead of the code above? I would try to avoid ignoring ts waring
```
withDefaults(
  defineProps<{
    value?: number | string | bigint | undefined
    dataCy?: string
    round?: number
  }>(),
  {
    value: '0',
    dataCy: '',
    round: 4,
  }
)
```

Reply:
tried this one before, but still throws an error. let me test it again

Reply:
yes, it still shows the error. lmk if you have another alternative

![image](https://github.com/kodadot/nft-gallery/assets/734428/e67b2273-c7ec-4179-af3d-e3bfc8d0f4df)

Reply:
Another alternative is here but there is not much difference.
<img width="466" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/ac5f5a85-a1fa-4568-acab-7865a385cd51">